src/utils/sourcify.ts

```diff
@@ -0,0 +1,10 @@
+export const lookUpContractWithSourcify = async ({ chainId, address }) => {
+  const baseUrl = `https://sourcify.dev/server/files/any`;

```

Comment:
Do you think we store this in env/configs/constants file?

Reply:
Sourcify will be used to lookup contracts that we are not storing ourselves. Or am I missing something in your question?

Reply:
Just wondering if we should pull this from config file, or env. Nothing serious

Reply:
Ah I see, now I understand!